server/base/src/main/java/org/apache/accumulo/server/util/checkCommand/SystemConfigCheckRunner.java

```diff
@@ -18,18 +18,91 @@
  */
 package org.apache.accumulo.server.util.checkCommand;
 
+import java.util.List;
+import java.util.Map;
+
+import org.apache.accumulo.core.Constants;
+import org.apache.accumulo.core.client.AccumuloException;
+import org.apache.accumulo.core.client.AccumuloSecurityException;
+import org.apache.accumulo.core.fate.AdminUtil;
+import org.apache.accumulo.core.fate.ZooStore;
+import org.apache.accumulo.core.fate.zookeeper.ZooReaderWriter;
+import org.apache.accumulo.core.lock.ServiceLock;
+import org.apache.accumulo.server.ServerContext;
+import org.apache.accumulo.server.cli.ServerUtilOpts;
 import org.apache.accumulo.server.util.Admin;
+import org.apache.zookeeper.KeeperException;
 
 public class SystemConfigCheckRunner implements CheckRunner {
   private static final Admin.CheckCommand.Check check = Admin.CheckCommand.Check.SYSTEM_CONFIG;
 
   @Override
-  public Admin.CheckCommand.CheckStatus runCheck() {
+  public Admin.CheckCommand.CheckStatus runCheck(ServerContext context, ServerUtilOpts opts,
+      boolean fixFiles) throws Exception {
     Admin.CheckCommand.CheckStatus status = Admin.CheckCommand.CheckStatus.OK;
+    printRunning();
+
+    System.out.println("\n********** Checking some references **********\n");
+    status = checkTableLocks(context, status);
```

Comment:
This check seems misplaced, maybe there should be a new Check.TABLE_LOCKS for this.

Reply:
Yeah, I was trying to avoid adding a new Check for at least a first round of reviews to avoid a larger diff/more refactoring. Added now. Made TABLE_LOCKS depend on SYSTEM_CONFIG, as that seemed like the most fitting location for it in the dependency tree.